docs/platforms/python/integrations/grpc/index.mdx

```diff
@@ -37,7 +37,7 @@ Select which Sentry features you'd like to install in addition to Error Monitori
 
 ### Server
 
-```python {"onboardingOptions": {"performance": "11-13", "profiling": "14-17"}}
+```python
```

Comment:
start/end comments are also missing here, it seems

Reply:
it seems like the current GRPC [client](https://docs.sentry.io/platforms/python/integrations/grpc/#client) onboarding has a bug in the ranges

I migrated it with what I felt is the correct ranges, please double check

Reply:
looks reasonable